### PR TITLE
Todo app refactor: first step

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

**Review by huszerlbence — COMMENTED** [review]
approved after color changes (highlight color and "completed" color).

nice one,
thanks

**Review by huszerlbence — APPROVED** [review]
Checked the modified files and found the hex code of the new color : )
thanks

File: examples/sn-react-tasklist/src/app.tsx [hunk 1]
```diff
@@ -1,26 +1,46 @@
 // start of material imports
-import { Container, CssBaseline, Grid } from '@material-ui/core'
+import { UniversalHeader } from '@sensenet/universal-header-react'
+import { Container, createStyles, CssBaseline, Grid, makeStyles } from '@material-ui/core'
 import Toolbar from '@material-ui/core/Toolbar'
 import React from 'react'
 // end of material imports
```
Comment (taki9) [inline examples/sn-react-tasklist/src/app.tsx:6]: remove the comments

File: examples/sn-react-tasklist/src/app.tsx [hunk 2]
```diff
@@ -1,26 +1,46 @@
 // start of material imports
-import { Container, CssBaseline, Grid } from '@material-ui/core'
+import { UniversalHeader } from '@sensenet/universal-header-react'
+import { Container, createStyles, CssBaseline, Grid, makeStyles } from '@material-ui/core'
 import Toolbar from '@material-ui/core/Toolbar'
```
Comment (taki9) [inline examples/sn-react-tasklist/src/app.tsx:4]: import this like the ones above

File: examples/sn-react-tasklist/package.json [hunk 3]
```diff
@@ -42,6 +42,7 @@
     "@sensenet/client-core": "^3.9.0",
     "@sensenet/default-content-types": "^4.0.0",
     "@sensenet/hooks-react": "^1.4.5",
+    "@sensenet/universal-header": "^1.0.0",
```
Comment (taki9) [inline examples/sn-react-tasklist/package.json:45]:
```suggestion
    "@sensenet/universal-header-react": "^1.0.0",
```

**Review by taki9 — CHANGES_REQUESTED** [review]
Remove sn-universal-header from packages. The merge messed it up.

**Review by taki9 — APPROVED** [review]

**Review by herflis — APPROVED** [review]

**Pull request merged**